source/documentation/modules/map.html.md.erb

```diff
@@ -49,8 +116,53 @@ title: sass:map
 <% end %>
 
 
+<% function 'map.set($map, $key, $value)',
+            'map.set($map, $keys..., $key, $value)',
```

Comment:
@nex3 Describing the signature like this (vs `map.set($map, $args...)`) massively simplifies the explanation. Is it worth reflecting this in the spec?

Reply:
The trouble is that this isn't valid Sass arglist declaration syntax (although arguably it should be). In order to use it in the spec, we'd need to formally specify exactly what it means and how it translates into actual behavior.

Even using it here worries me a little because it implies that `$key` and `$value` can be passed by name in the second signature when in fact they can't. It may still be worth it with a note explaining that, though.

Reply:
Oof, right, I forgot that arglists always have to follow positional args. Maybe I'll change this signature back to `$args`, but then "deconstruct" it as `$map, $keys..., $key, $value` in the explanation